toolkit/tools/imagegen/configuration/systemconfig.go

```diff
@@ -176,8 +176,6 @@ func (s *SystemConfig) IsValid() (err error) {
 
 	//Validate Encryption
 
-	//Validate HidepidDisabled
```

Comment:
More of a high level discussion, do we want to start adding logic to validate migrations? I'm not sure we can do much here, but at a high level I'm thinking it might be interesting to detect if a user is setting `HidepidDisabled` and mark it as deprecated somehow...

Outside the scope of this PR I think, but might be an interesting topic as we start to make larger changes to the config formats.

Reply:
Interesting point - this can be a tool feature we can discuss further outside the PR. Feature issue created - #6973